rrfs-test/testinput/rrfs_fv3jedi_hyb_2022052619.yaml

```diff
@@ -90,10 +90,11 @@ cost function:
                 alias:
                 - in code: common
                   in file: fixed_250km_0.3
-
               drivers:
                  multivariate strategy: duplicated
                  read local nicas: true
+              model:
+                 level for 2d variables: last
```

Comment:
Just curious, what does this mean?  @TingLei-NOAA @SamuelDegelia-NOAA

Reply:
Per @TingLei-NOAA - This tells bump to treat the 2d variables as the last level in a 3d structure data, which is needed since the fset is stored from top to bottom when passed to saber.  This option is not needed for this ctest since no 2d analysis variable is used like surface pressure. But adding it here would avoid possible future overlooking on this issue.

Reply:
Yes, to my understanding `fset` defines the fields used by saber.

Reply:
Thanks!